extensions/amp-user-notification/0.1/test/test-amp-user-notification.js

```diff
@@ -92,11 +102,36 @@ describes.realWin('amp-user-notification', {
   it('should NOT require `data-show-if-href`', () => {
     const el = getUserNotification({
       id: 'n1',
+      'data-show-if-geo': 'nafta',
+    });
```

Comment:
nit: shall we add a test without any `data-show-if-*`

Reply:
I'll add one